modules-themes/readme.md

```diff
@@ -0,0 +1,42 @@
+# Listing third party open source and free of charge modules and themes for the PrestaShop e-commerce solution
+
+There are plenty of useful themes and modules available on the web from third party developers and company for the PrestaShop e-commerce solution. But unfortunately, there is no place currently to list them and find them.
+
+So, let's build a knowledge base. If you want to contribute to this page, please create a pull request with your modules and themes, following the example:
+
+**Module/theme name**
+* short description (one or two lines max)
+* repository URL (like a GitHub repository link)
+* download page link
+* keywords, topics, features, separated by a comma
+
+This will also be used as a base to create content for a dedicated website in the future.
+
+<hr />
+
+## Modules
+
+**Console for PrestaShop 1.7**
+* A Command line tool that better supports PrestaShop legacy classes using the current PrestaShop Console and a list of useful commands that you can use and reuse for learning purposes
+* https://github.com/friends-of-presta/fop_console
+* https://github.com/friends-of-presta/fop_console/releases
```

Comment:
I guess a single link per project would suffice?

Reply:
what if people don't provide a zip on the repo, but rather ask to come on their download page?

Reply:
Usually repositories have instructions on how to get the project releases. Many projects are distributed through platforms like `npm` or `brew`. 

We can reduce the noise by adding just one link by default (we can add more if necessary)

Reply:
npm or brew is ok for devs.
I am also thinking about non devs users.

Reply:
That's just an example. Usually the repository readmes explain how to get the release, regardless of who the public is. If the project has a website, we can use the website instead of the repository.

Reply:
actually, I checked many repos, and no, it's not explained most of the time. and there is no release with a ready to use zip.

setting this kind of prerequisite would help end users, and will make aware devs from the need